python/src/opendp/context.py

```diff
@@ -212,7 +212,10 @@ def metric_of(M) -> Metric:
     raise TypeError(f"unrecognized metric: {M}")
 
 
-def loss_of(epsilon=None, delta=None, rho=None, U=None) -> Tuple[Measure, float]:
+LossParameter = float | Tuple[float, float]
```

Comment:
I dont think this will work before 3.10 without `from __future__ import annotations`?